### PR TITLE
[site] nav bar edits

### PR DESCRIPTION
**Pull request by mkveerapen**

### CODE REVIEW

**Comment by johnc1231** [comment]
@mkveerapen You should assign this to someone so they know to review it. For website stuff, pick an 8th floor person.

**Comment by johnc1231** [comment]
It seems like you're deleting the link to the 0.1 docs though, which I don't think we should do.

**Comment by mkveerapen** [comment]
I was talking to @tpoterba  about removing the 0.1 docs because the (possibly) last user on Earth (me) has switched over to 0.2! So, I deleted the link to 0.1 to make it cleaner

**Comment by johnc1231** [comment]
I have two responses to that:

1. I am willing to bet there's some other user somewhere, and while I wish they would stop they may still be doing it. 
2. More importantly, I suspect research papers exist that reference Hail 0.1 code. Someone reading a paper with published code should be able to look at the documentation of that code.

**Comment by johnc1231** [comment]
I'd be ok with taking it off the nav bar if there was some other link to it that was reasonable and findable. Python is a good example. They always show newest thing, but they have a drop down at the top that lets you switch: https://docs.python.org/3/library/re.html

**Comment by mkveerapen** [comment]
Fair. Perhaps we could put it in a separate link elsewhere separate from the top ribbon?

**Comment by mkveerapen** [comment]
Let's put it at the footer of the front page?

**Comment by johnc1231** [comment]
I think it should at least be somewhere connected to documentation. Maybe somewhere on this page: https://hail.is/docs/0.2/index.html

**Comment by mkveerapen** [comment]
Love that idea!

**Comment by mkveerapen** [comment]
This line at the end of that page

If you would like to refer to our Hail v0.1 (deprecated) docs, please view `Hail 0.1 docs <https://hail.is/docs/0.1/index.html>`

File: hail/www/navbar.html [hunk 1]
```diff
@@ -51,8 +31,14 @@
       <li id="code" class="nav-item">
         <a href="https://github.com/hail-is/hail">CODE</a>
       </li>
-      <li id='workshops' class="nav-item">
-        <a href="https://workshop.hail.is">WORKSHOPS</a>
+      <li id='powered-science' class="nav-item">
+        <a href="https://hail.is/references.html">POWERED-SCIENCE</a>
+      </li>
+      <li id='blog' class="nav-item">
+        <a href="https://blog.hail.is/">HAIL</a>
```
Comment (johnc1231) [inline hail/www/navbar.html]: Why is the blog called `HAIL`. Should it not be `BLOG`?

**Pull request merged**